src/jobflow_remote/config/base.py

```diff
@@ -166,6 +166,13 @@ class WorkerBase(BaseModel):
         None,
         description="String with commands that will be executed after the execution of the Job",
     )
+    execution_cmd: Optional[str] = Field(
+        None,
+        description="String with commands to execute the Job on the worker. By default will be "
+        "set to `jf -fe execution run {}`. The `{}` part will be used to insert the path to "
```

Comment:
If you set the default to a value different from `None` and you generate a project with something like `jf project generate`, or create it from the python interface and dump it, the value will be explicitly set in the yaml file. I see two downsides there: 1) making the config a bit more cramped 2) this is a setting that it is extremely unlikely to be changed. If for any reason we need to make a change to the default in the future, this is a potential source for backward incompatibility issues. I know we prepared the upgrade procedure, but if we can reduce the amount of times this needs to be used it would be better. So I decided to set the default to `None` and describe the default in the doc. 

For the validation, I had also considered and not added it for the usual fear of overloading the process of the Project validation process. Again, since this is a very unlikely option to be set I imagined that this will almost always be a wasted check. It is also true that if the default value is None there is also nothing to check, so the overhead should be negligible. So it can still be added. In that case I would just check that if it is not `None` there should be a `{}` inside the string. Or should it contain the full string `jf -fe execution run {}`? I wonder if there might be cases where one needs to write the pieces of the command with some kind of separator.

Reply:
I see, good point to keep it outside then. You can add the validation if it's not None as you mention, I think it may help (and it should be easy to implement right ?) : there should be one and only one {} inside the string. (I would indeed not go the the fact that it shoud contain the full string).